scripts/femm/readme.txt

```diff
@@ -0,0 +1,19 @@
+`mirror.FEM` can be loaded by the FEMM software
```

Comment:
Awesome! Can you please combine this with the `README.md` in the same dir?
That will just render nicer when someone opens the dir on GitHub.

Can you note the FEMM version that was used, so we remember in a few years?

Reply:
@RemiLehe I pushed a commit that fixes that. Can you pls take a look?